WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -35,6 +35,21 @@ class PostsCardViewModel: NSObject {
 
     private weak var viewController: PostsCardView?
 
+    lazy var diffableDataSource = UITableViewDiffableDataSource<Int, NSManagedObjectID>(tableView: viewController!.tableView) { [weak self] (tableView, indexPath, objectID) -> UITableViewCell? in
```

Comment:
[nit] You could add a couple of type aliases to clean up the code a bit:
```
typealias DataSource = UITableViewDiffableDataSource<Int, NSManagedObjectID>
typealias Snapshot = NSDiffableDataSourceSnapshot<Int, NSManagedObjectID>
```

Reply:
Done in 3b3dfd6b4e033ec8be565623fb761af71777b256